src/pocketmine/lang/locale/eng.ini

```diff
@@ -213,13 +223,14 @@ commands.xp.failure.withdrawXp=Cannot give player negative experience points
 commands.xp.success=Given {%0} experience to {%1}
 commands.xp.success.levels=Given {%0} levels to {%1}
 commands.xp.success.negative.levels=Taken {%0} levels from {%1}
-commands.xp.usage=/xp <amount> [player] OR /xp <amount>L [player]
+commands.xp.usage=/xp <amount> [player] OR /xp <amount of levels> [player]
```

Comment:
`OR /xp <amount of levels> [player]` - why you removed `L`? This must be a `/xp <amount of levels>L [player]`.